projects/phone/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './lib/masks';
```

Comment:
Should we narrow public API ?

For example, i think that `TEMPLATE_FILLER` should not be in public API

Reply:
Should processors be in public API?